### PR TITLE
August changes

### PR DESCRIPTION
**Pull request by AnhPhantom**

### CODE REVIEW

File: parsing_2.py [hunk 1]
```diff
@@ -85,13 +85,38 @@ def main():
...
+    #that number is stored in max_requests and the name in max_requests_name
+    max_requests = 0
+    max_requests_name = 'start'
+    for request in requests_dict:
+        if requests_dict[request] > max_requests:
+            max_requests = requests_dict[request]
+            max_requests_name = request
+
+
+    total_responses = len(log_lines)
```
Comment (PatchMixolydic) [inline parsing_2.py:112]: This appears to be unused

**Review by PatchMixolydic — COMMENTED** [review]
Thank you! Just one tiny nitpick

File: parsing_2.py (hunk 1 above)
Comment (AnhPhantom) [reply parsing_2.py:112]: I use line 112 it to calculate the percentage of redirect and error

Comment (PatchMixolydic) [reply parsing_2.py:112]: My bad ^^;

**Review by PatchMixolydic — APPROVED** [review]

File: parsing_2.py [hunk 2]
```diff
@@ -85,13 +85,38 @@ def main():
...
     print(f"Between {first_date_str} and {last_date_str}, there were {len(log_lines)} requests made to our website")
     print(f"In the last six months ({last_six_months_str} - {last_date_str}), there were {len(last_six_months)} requests made to our website")
     print("Total number of redirects:", Redirect_count)
     print("Percentage of redirect request: {0:.1%}".format(Redirect_count/total_responses))
     print("Total number of Errors:", Error_count)
     print("Percentage of error request: {0:.1%}".format(Error_count/total_responses))
+    print("The most requested file: {max_requests_name}")
```
Comment (PatchMixolydic) [inline parsing_2.py:119]: I missed something; this should be
```python
    print(f"The most requested file: {max_requests_name}")
```
(note the `f` before the string, which enables string interpolation)

**Pull request merged**